spec/amp-cache-guidelines.md

```diff
@@ -56,6 +56,27 @@ Provide guidelines to external parties for implementing their own AMP Caches.
 
 * Participation in [AMP Ads For AMP Pages ("A4A")](https://github.com/ampproject/amphtml/issues/3133) advertisement signing.
 
+## Guidelines: Accepted MIME types
+
+### MIME types for images
+
+Accepted MIME types for images include all `image/` subtypes (e.g. `image/gif`).
+
+### MIME types for fonts
+
+Accepted MIME types for fonts include all `font/` subtypes (e.g. `font/opentype`), as well as the following types.
+
+| Media type / subtype  |
+| ------------- |
+|  application/font   |
```

Comment:
Good call—updated. Can potentially simplify by not using the wildcard notation and example, but seemed helpful.